src/store/actions/drinkCategories.js

```diff
@@ -14,9 +14,6 @@ export const fetchDrinkTypes = () => {
       .get('https://www.thecocktaildb.com/api/json/v1/1/list.php?c=list')
       .then(response => {
         dispatch(setFetchedTypes(actions.GET_DRINK_TYPES, response.data.drinks));
-      })
-      .catch(error => {
-        dispatch(console.log(error));
```

Comment:
why did you delete the "catch" here and below, i know it have a console.log but then lets handle this in another way but the catch are pretty much necessary, if you only deleted this because of the console.log, please let me know